docs/design/add-in-icons-monoline.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Monoline style icon guidelines for Office Add-ins
+description: ''
+ms.date: 12/09/2019
+localization_priority: Priority
+---
+
+# Monoline style icon guidelines for Office Add-ins
+
+Monoline style iconography are used in Office 365. If you would prefer that your icons match the Fresh style of non-subscription Office 2013+, see [Fresh style icon guidelines for Office Add-ins](add-in-icons-fresh.md).
+
+## Office Monoline visual style
+
+The goal of the Monoline style to have consistent, clear, and accessible iconography to communicate action and features with simple visuals, ensure the icons are accessible to all users, and have a style that is consistent with those used elsewhere in Windows.
+
+The following guidelines are for 3rd party developers who want to create icons for features that will be consistent with the icons already present Office products.
+
+### Design principles
+
+-   Simple, clean, clear.
+-   Contain only necessary elements.
+-   Inspired by Windows icon style.
+-   Accessible to all users.
+
+#### Conveying meaning
+
+-   Use descriptive elements such as a page to represent a document or an envelope to represent mail.
+-   Use the same element to represent the same concept, i.e. mail is always represented by an envelope, not a stamp.
```

Comment:
Can also spell out instead of using abbreviation

```suggestion
-   Use the same element to represent the same concept, i.e., mail is always represented by an envelope, not a stamp.
```